module/bookCollection.js

```diff
@@ -0,0 +1,25 @@
+class BookCollection {
+  constructor() {
+    this.collectionData = JSON.parse(window.localStorage.getItem('collectionArray')) || [];
+    this.removeButtons = [];
+  }
+
+  addBook(book) {
+    this.collectionData.push(book);
+    this.updateStorage();
+  }
```

Comment:
- [ ] When I'm writing a title and author I can't see what I'm typing this may confuse the user if he/she's really typing so please make sure that text is visible to the user when typing 
|
<img width="585" alt="Screenshot 2023-02-22 at 15 31 42" src="https://user-images.githubusercontent.com/61790587/220653328-b0baf504-cb92-48e5-bf6d-a7f573fd07ed.png">